dash.el

```diff
@@ -1591,6 +1591,54 @@ and when that result is non-nil, through the next form, etc."
                    (--> ,result ,form))
                  ,@more))))
 
+(defmacro -cond-> (x &rest clauses)
+  "Conditionally thread X through CLAUSES.
+Threads x (via `->') through each form for which the
+corresponding test expression is true.  Note that, unlike cond
+branching, `-cond->' threading does not short circuit after the
+first true test expression."
+  (declare (debug (form body))
+           (indent 1))
+  (when (-> clauses length (% 2) (= 1))
```

Comment:
This is checking that the number of clauses is odd, right? Shouldn't it be even instead?

[Not that my vote means much, but I'd vote for writing in simple Elisp rather than Clojure where the latter doesn't significantly simplify things, such as in this one-liner.]

Reply:
It does check if the number of clauses is odd, in which case there is an error, i.e there is a test with no expression or vice-versa. I guess there is no problem with using `(unless ...)`.

[This one liner maybe a vice of mine hahaha]

Reply:
Oops, got mixed up. I wasn't suggesting switching the `when` to `unless`, FTR.